docs/zephyr/quickref.rst

```diff
@@ -36,7 +36,10 @@ Use the :ref:`machine.Pin <machine.Pin>` class::
 
     from machine import Pin
 
-    pin = Pin(("GPIO_1", 21), Pin.IN)   # create input pin on GPIO1
+    gpio1 = "gpio@400ff040"             # GPIO1 device name
+    gpio2 = "gpio@400ff080"             # GPIO2 device name
```

Comment:
This seems quite.... un-user-friendly, to know the address.  Is there a better solution?  How do other Zephyr projects handle this?  Maybe in the device tree / overlay we can have an alias?

Reply:
As @jimmo says below, he'll look into this.

Reply:
Please have a look at #10859